force-app/main/default/classes/ElevateBatchItemCreateResponse.cls

```diff
@@ -0,0 +1,260 @@
+ *     "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ *     LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ *     FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ *     COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ *     INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ *     BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ *     LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ *     CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ *     LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ *     ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ *     POSSIBILITY OF SUCH DAMAGE.
+ */
+public class ElevateBatchItemCreateResponse {
+    private List<Errors> errors;
+    public ElevateBatchItemType batchItemType;
+    public CommitmentResponse commitmentResponse;
+    public PurchaseResponse purchaseResponse;
+
+    public class Schedules {
+        public Integer amount;
+        public String canceledAt;
+        public String createdAt;
+        public String currencyCode;
+        public String firstOccurrenceOnTimestamp;
+        public String frequency;
+        public Integer frequencyInterval;
+        public String id;
+        public String merchantId;
+        public String modifiedAt;
+        public Integer nextPaymentAmount;
+        public Date nextPaymentDate;
+        public String nextPaymentTimestamp;
+        public String previousId;
+        public String recentTransactionId;
+        public String status;
+    }
+
+    public class CommitmentResponse {
+        public String accountName;
+        public AchData achData;
+        public String addressLine1;
+        public String addressLine2;
+        public Integer amountPaidToDate;
+        public String batchId;
+        public String canceledAt;
+        public CardData cardData;
+        public String city;
+        public String commitmentType;
+        public String country;
+        public Datetime createdAt;
+        public String currencyCode;
+        public String email;
+        public String firstName;
+        public String gatewayId;
+        public String id;
+        public String importRefId;
+        public String lastName;
+        public String merchantId;
+        public String merchantName;
+        public Metadata metadata;
+        public String paymentMethodType;
+        public String phone;
+        public String postalCode;
+        public Metadata productMetadata;
+        public String productMetadataSchemaUri;
+        public String salutation;
+        public List<Schedules> schedules;
+        public String state;
+        public String status;
+        public String statusReason;
+        public String statusReasonComment;
+        public String suffix;
+        public Integer version;
+    }
+
+
+    public class Metadata {
```

Comment:
Will this be updated and used later?

Reply:
This could contain key/value pairs if we have passed product metadata in the request so I left it here since it could potentially be part of the response we receive